magda-esri-portal-connector/src/EsriPortal.ts

```diff
@@ -474,44 +474,48 @@ export default class EsriPortal implements ConnectorSource {
                 item.distributions.push(wmtsDist);
             }
         } catch (err) {
-            console.log(`Broke on item url: ${item.url}, dist uri: ${distUri}`);
-            console.log(err);
-        }
-    }
-
-    private async processLayerAsDistribution(
-        lyr: any,
-        item: any,
-        distUri: any
-    ) {
-        try {
-            const lyrUrl = distUri
-                .clone()
-                .segment(lyr.id.toString())
-                .toString();
-            const subDistInfo = await this.requestDistributionInformation(
-                lyrUrl
+            console.error(
+                `Broke on item url: ${item.url}, dist uri: ${distUri}`
             );
-            if (subDistInfo.error) return;
-            const subDist = {
-                accessURL: lyrUrl,
-                title: subDistInfo.name,
-                name: subDistInfo.name,
-                description: subDistInfo.description,
-                type: `Esri ${subDistInfo.type}`,
-                id: `${item.id}-c${lyr.id}`
-            };
-            item.distributions.push(subDist);
-        } catch (err) {
-            console.log(
-                `Broke on item url: ${
-                    item.url
-                }, dist uri: ${distUri}, layer id: ${lyr.id}`
-            );
-            console.log(err);
+            console.error(err);
         }
     }
 
+    // private async processLayerAsDistribution(
+    //     lyr: any,
+    //     item: any,
+    //     distUri: any
+    // ) {
+    //     try {
+    //         const lyrUrl = distUri
+    //             .clone()
+    //             .segment(lyr.id.toString())
+    //             .toString();
+    //         const subDistInfo = await this.requestDistributionInformation(
+    //             lyrUrl
+    //         );
+    //         if (subDistInfo.error) {
+    //             return;
+    //         }
+    //         const subDist = {
+    //             accessURL: lyrUrl,
+    //             title: subDistInfo.name,
+    //             name: subDistInfo.name,
+    //             description: subDistInfo.description,
+    //             type: `Esri ${subDistInfo.type}`,
+    //             id: `${item.id}-c${lyr.id}`
+    //         };
+    //         item.distributions.push(subDist);
+    //     } catch (err) {
+    //         console.error(
+    //             `Broke on item url: ${
+    //                 item.url
+    //             }, dist uri: ${distUri}, layer id: ${lyr.id}`
+    //         );
+    //         console.error(err);
+    //     }
+    // }
+
```

Comment:
I left this here because the logic of the connector changed and we still may want to do this at some stage, basically it's collecting additional distributions associated with a dataset...

Happy to cull it though if that's your preference. Or I could at least add some comments as to why it's there.

Reply:
Yeah happy to leave it in as long as there's a comment :)